src/main/java/net/tridentsdk/entity/meta/living/animal/RabbitMeta.java

```diff
@@ -0,0 +1,17 @@
+package net.tridentsdk.entity.meta.living.animal;
+
+/**
+ * @author TridentSDK
+ * @since 0.5-alpha
+ */
+public interface RabbitMeta extends AnimalMeta {
+
+    RabbitType getRabbitType();
+
+    void setRabbitType(RabbitType type);
+
+    enum RabbitType {
```

Comment:
Break out as it's own file, populate with information from http://minecraft.gamepedia.com/Rabbit#Data_values